app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -144,6 +144,18 @@ window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion;
     var action = expanded ? 'accordionOpened' : 'accordionClosed'
     var options = { transport: 'beacon', label: label }
 
+    // optional parameters are added to the parent
+    // heading not the button that is clicked
+    var extraOptions = section.parentElement && section.parentElement.getAttribute('data-track-options')
```

Comment:
Just for the reference of anyone else reviewing this and wondering why we're creating an object for just one attribute: I think we have a convention to set page scoped dimensions (eg. category) as their own data attributes on the element and then create an options object for any event scoped ones. We usually do this for links I think and this is a heading but it makes sense to follow the same convention here.